omega/omega-connector/omega-connector-grpc/src/main/resources/ssl.properties

```diff
@@ -13,5 +13,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ciphers = ECDHE-RSA-AES128-GCM-SHA256,ECDHE-RSA-AES256-GCM-SHA384,ECDHE-ECDSA-AES128-SHA256
+ciphers = ECDHE-RSA-AES128-GCM-SHA256,ECDHE-RSA-AES256-GCM-SHA384
```

Comment:
It's better to add a link of the context why do we remove the cipher to help others trace the issue.

Reply:
I added the link in the comment of the commit. Do I still need to explain it in the source file?